tests/test_trace.py

```diff
@@ -86,12 +86,8 @@ def test_trace_ip():
             validate_schema(obj)
 
 
-def test_non_existant_event_type_validation_rejection(caplog):
-    """
-    Test if an incoming trace with a non-supported schema logs a warning
-    instead of printing entire stack-trace
-    """
+def test_non_existant_event_type_validation_rejection():
```

Comment:
Why was the docstring removed here?

Reply:
Because it doesn't reflect what the test does anymore, and the new version of the test is clear enough to not need a docstring (in my opinion)